### DISCUSSION TITLE
Alexa Media Player causa frequenti disconnessioni con Google Home?

### DISCUSSION THREAD

**Discussion by Vladnetwa** [indomus/forum#2878] [HUB: Home Assistant]
Un saluto a tutta la community.
Ho recentemente installato Home Assistant su una Vm con Hiper-V e con le guide reperibili sul sito ho potuto integrare la connessione con i dispositivi Google Home e Elexa/Echo presenti nell'appartamento, grazie alle quali è possibile gestire alcuni dispositivi con i comandi vocali . Fin qui tutto bene.  
Nell'ultimo passaggio ho eseguito  l'installazione del modulo Alexa Media Player. Da qui mi sorge un dubbio.
Nel momento in cui HA ha in funzione questo modulo, sto riscontrando che nell'app Google Home i dispositivi presentati da Home Assistant si disconnettano molto frequentemente.
Anche disattivando momentaneamente Alexa Media Player  i dispositivi sono frequentemente disconnessi, costringendomi a disconnettere e riconnettere l'account di Home Assistant dall'app Google Home. Prima che installassi questo modulo i dispositivi erano sempre presenti.
Capita anche a voi?

**Comment by indomus** [comment]
non c'è alcuna correlazione, e te lo conferma il fatto che disabilitando alexa media player il problema sussiste.

**Reply by Vladnetwa** [reply]
Allora sarà una coincidenza. Ricontrollerò meglio....magari me lo tengo qualche giorno senza l'integrazione.....Grazie mille.
